tests/unit/EntityInfo.spec.ts

```diff
@@ -131,4 +134,84 @@ describe('EntityInfo', () => {
     expect(wrapper.find('#nr-subtitle').exists()).toBeFalsy()
     expect(wrapper.find('#nr-number').exists()).toBeFalsy()
   })
+
+  it('displays the breadcrumb correctly', async () => {
+    const store = getVuexStore()
+    const router = mockRouter.mock()
+
+    sessionStorage.clear()
+    sessionStorage.setItem('TEMP_REG_NUMBER', 'T123456789')
+
+    store.state.entityName = 'My Named Company'
+    const wrapper = mount(EntityInfo, { vuetify, store, router })
+
+    await Vue.nextTick()
+
+    expect(wrapper.find('.breadcrumb').exists()).toBe(true)
+    expect(wrapper.find('.breadcrumb').text()).toContain('My Account')
```

Comment:
I thought we were going with "Manage Businesses" for the first level.

Also, if would be great if you could check each level individually instead of checking that the who thing "contains" the separate crumbs.

Reply:
'My account' does go to Manage businesses. The dashboard name is the business name.
You will always have the My Account and Dashboard level. It only goes one level deeper, and those are all checked individually.

Reply:
See comment in epic ticket
https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/3740

Reply:
Very Confusing.. 

https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/3763

This ticket says see the wireframe above which shows my account.. ?

Reply:
https://preview.uxpin.com/b3e60baaf1f0418fec3fac72fcc7bdce6f772e13#/pages/128469428/specification/no-panels

Reply:
Maybe ask Scott for the latest decision on this?

Reply:
Way ahead of ya :)

Reply:
Updated!